keras_preprocessing/image/utils.py

```diff
@@ -127,10 +127,21 @@ def load_img(path, grayscale=False, color_mode='rgb', target_size=None,
     return img
 
 
-def list_pictures(directory, ext='jpg|jpeg|bmp|png|ppm'):
+def list_pictures(directory, ext=('jpg', 'jpeg', 'bmp', 'png', 'ppm')):
```

Comment:
tif and tiff?

Reply:
it has been this way since the beginning https://github.com/keras-team/keras-preprocessing/blame/014457b52def2af30bff88809437f8941451b76f/keras_preprocessing/image.py#L542
and I'm not sure we should change it since there is no issue related as we can break some systems with the default behavior. What do you think?

Reply:
it has been this way since the beginning https://github.com/keras-team/keras-preprocessing/blame/014457b52def2af30bff88809437f8941451b76f/keras_preprocessing/image.py#L542
and I'm not sure we should change it since there is no issue related as we can break some systems with the default behavior. What do you think?

Reply:
Iterator accepts tif and tiff images, I don't see any reason why it should be left out here and also list_pictures function is not used in any internal code, so changing it doesn't going to break anything I believe.

Reply:
I understand and agree with your point of being used in Iterator but I meant non-internal code but code of others. 
But sure let's add it. @jnphilipp

Reply:
Has anyone a tiff image to test if ``load``` and ```img_to_array``` work. If so I would add it, if not then not.

Reply:
please add the `tif` and `tiff` extension